setup/default.xml

```diff
@@ -33,7 +33,7 @@
     <entry key='database.generateQueries'>true</entry>
 
     <entry key='database.changelog'>./schema/changelog-master.xml</entry>
- 
```

Comment:
This file is still in the diff. Please check diff yourself before saying that it's fixed.